cosmovisor/cmd/cosmovisor/cmd/help.go

```diff
@@ -49,5 +38,10 @@ documented in: https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor/READM
 
 To get help for the configured binary:
   cosmovisor run help
+
+Available Commands:
+  help     This help message
+  run      Runs app passing all subsequent parameters
+  version  Prints version of cosmovisor and the associated app.
```

Comment:
Lines 39-40 can probably be removed now that this is added.

Reply:
Why? I think it provides more help. We don't show the list of commands otherwise.

Reply:
I misread. I thought it was a duplication of the help command but I see now that it is separate.